### PR TITLE
Readme Overhaul

### PR DESCRIPTION
**Pull request by not-nef**
What do you think? Suggestions are welcomed and wanted!

### CODE REVIEW

**Comment by im-coder-lg** [comment]
Excellent!

**Comment by im-coder-lg** [comment]
I say this is ready for review!

**Comment by im-coder-lg** [comment]
Also, before the word `release` in the SVG, can you try adding `Latest`? It _might_ make users clear that we are giving the latest release.

**Comment by sumeshir26** [comment]
@not-nef @im-coder-lg I still think there should be a banner to inform users about new releases.

**Comment by im-coder-lg** [comment]
Yeah even me... Why not the banner be on the website? I mean, that's where users would head for documentation, a gist of features, and etc.
Maybe... We'll enlarge the SVG sizing and maybe modify the size a bit.

**Review by im-coder-lg — DISMISSED** [review]
SVG sizing and maybe text modification. What's your idea?

**Comment by sumeshir26** [comment]
@not-nef Maybe we can add more badges through this: https://github.com/Ileriayo/markdown-badges

**Comment by im-coder-lg** [comment]
Maybe these badges:
```markdown
![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
![Gitpod](https://img.shields.io/badge/gitpod-f06611.svg?style=for-the-badge&logo=gitpod&logoColor=white)
![Vercel](https://img.shields.io/badge/vercel-%23000000.svg?style=for-the-badge&logo=vercel&logoColor=white)
![GitHub Actions](https://img.shields.io/badge/githubactions-%232671E5.svg?style=for-the-badge&logo=githubactions&logoColor=white)
```
These are the ones that seem related for us; any ideas?

**Comment by im-coder-lg** [comment]
We could make the SVGs into links, so I could try linking them like a suggestion.

**Comment by sumeshir26** [comment]
This one as well:
`Repl.it`

**Comment by im-coder-lg** [comment]
I thought of it, well let's add them!

**Comment by not-nef** [comment]
ok

**Comment by not-nef** [comment]
it looks terrible @im-coder-lg. i wouldnt use those and just leave the release bade there. But you guys say what to do.

**Comment by not-nef** [comment]
image

i mean cmon it just looks trash with that small release badge. But i say we definitely need to keep the release badge and tbh i dont think the others are that important.

**Comment by sumeshir26** [comment]
@not-nef Let me try to make a custom release badge

**Comment by im-coder-lg** [comment]
Wait are you designing a SVG?

**Comment by im-coder-lg** [comment]
> image

@not-nef that's what we need. 
> i mean cmon it just looks trash with that small release badge. But i say we definitely need to keep the release badge and tbh i dont think the others are that important.

Hold up! You are using **HTML**! Just use a `width` attribute inside your `` tag!

**Comment by sumeshir26** [comment]
@im-coder-lg Yours is a much better idea

**Comment by im-coder-lg** [comment]
@not-nef seems to be confused. I think 50px is a better idea. I will give the code here, all you need to do is replace and commit.
Don't get confused, it's so easy.  Just **copy the line**, replace it on Line 8 and commit.
Line 8:
```HTML
        <img src="https://img.shields.io/github/release/sumeshir26/TimerX.svg?style=flat-square" alt="Releases" />
```
To:
```HTML
        <img src="https://img.shields.io/github/release/sumeshir26/TimerX.svg?style=flat-square" alt="Releases" width="50px" />
```

**Comment by not-nef** [comment]
@im-coder-lg @sumeshir26 what do you guys think of the new Contributing file?

**Comment by im-coder-lg** [comment]
Never noticed it... I will inspect it to fix grammatical issues. Also #41 needs urgent merging!

**Comment by im-coder-lg** [comment]
I will manage the Contributing guide. Do I need write access to edit since it's on your fork?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Contribute to TimerX
+
+TimerX is written in **Python**, so you have to install it if you want to contribute.
+You can download  the latest version [here](https://www.python.org/downloads/release/python-3101/).
```
Comment (im-coder-lg) [inline CONTRIBUTING.md]: There's even Anaconda3 which adds Python to your PATH, does the work for you. Moreover, old versions of Python mean it's easier to manage, right?

**Comment by im-coder-lg** [comment]
The contributing guide is kinda messed up. It's haywire, like my method. But don't you worry. I will fix things up. I was waiting to make this, you did! I will make the same file over at the website too!

**Comment by not-nef** [comment]
cool

**Comment by im-coder-lg** [comment]
Updated the contributing guide. Made sure the base was yours, I added some extras. Maybe I'll make it easy for new people to contribute here.

**Comment by im-coder-lg** [comment]
Now, I can approve.

**Review by im-coder-lg — APPROVED** [review]
lasgtm(looks _and_ seems good to me)

**Comment by im-coder-lg** [comment]
@sumeshir26 your opinions needed! PS: Some new PRs opened, please inspect)

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-</div>
+<p align="center">
+    <img alt="TimerX Logo" src="./assets/logo_new.png" width="100px" />
+    <h1 align="center">TimerX</h1>
+</p>
 
-# <img alt="TimerX icon" width="32px" src="https://raw.githubusercontent.com/sumeshir26/TimerX/master/assets/logo_new.png"> TimerX
+<p align="center">
+    <a style="text-decoration:none" href="https://github.com/sumeshir26/TimerX/releases">
+        <img src="https://img.shields.io/github/release/sumeshir26/TimerX.svg?style=flat-square" alt="Releases" />
```
[the remark this answers is not in the document]
Comment (not-nef) [reply README.md]: nah, 50 is too small. With this size it perfectly blends in.

[the remark this answers is not in the document]
Comment (not-nef) [reply README.md]: oh wait this was made before i did that change sorry

File: CONTRIBUTING.md (hunk 1 above)
Comment (not-nef) [reply CONTRIBUTING.md]: dunno, i would leave it as it is

**Comment by not-nef** [comment]
@im-coder-lg i think we should get rid of the todo list since i dont think it belongs in the readme. if we keep it i think we should put it in another document.

**Review by sumeshir26 — APPROVED** [review]

**Comment by sumeshir26** [comment]
Should I merge?

**Review by im-coder-lg — APPROVED** [review]
lgtm

**Pull request merged**